homeassistant/components/plex/server.py

```diff
@@ -201,3 +220,7 @@ def playlist(self, title):
     def create_playqueue(self, media, **kwargs):
         """Create playqueue on Plex server."""
         return plexapi.playqueue.PlayQueue.create(self._plex_server, media, **kwargs)
+
+    def url(self, key, includeToken=False):
```

Comment:
Do we still need this?

Reply:
Yes. It's a pass through to the attribute object.